packages/kbn-expandable-flyout/src/context.tsx

```diff
@@ -82,7 +111,7 @@ export const ExpandableFlyoutProvider = ({ children }: ExpandableFlyoutProviderP
       left?: FlyoutPanel;
       preview?: FlyoutPanel;
     }) => dispatch({ type: ActionType.openFlyout, payload: { left, right, preview } }),
-    [dispatch]
```

Comment:
Why remove the dispatch?

Reply:
! thanks, its not intentional

Reply:
in fact, `dispatch` is stable, so I guess I can remove it everywhere lol